repos/system_upgrade/el7toel8/actors/checklocalnetworkscripts/libraries/library.py

```diff
@@ -0,0 +1,26 @@
+import os
+from leapp import reporting
+
+
+def generate_report():
+    """ Generate reports informing user about possible manual intervention requirements """
+
+    DOC_URL = "https://red.ht/upgrading-RHEL-8-to-RHEL-9-network-scripts"
```

Comment:
The document is invalid now. I guess it will not be available until the RHEL 9 release and this actor is now in the repository for the upgrade RHEL 7 to RHEL 8. What's the expectation? In case it's just for IPU 8 -> 9, it should be in the `el8toel9` repository, not `el7toel8`.

Reply:
Thanks, I moved to `el8toel9` dir.